core/transport/src/main/proto/grpc/meta_master.proto

```diff
@@ -224,6 +224,14 @@ message GetMasterIdPResponse {
   optional int64 masterId = 1;
 }
 
+message GetClusterIdInternalPOptions {}
+message GetClusterIdInternalPRequest {
+  optional GetClusterIdInternalPOptions options = 1;
+}
+message GetClusterIdInternalPResponse {
+  optional string clusterId = 1;
+}
```

Comment:
when does the client need to call this? I thought the clusterID is returned in GetWorkerIdPResponse?